SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp

```diff
@@ -83,7 +99,9 @@ void UGeneratedTestMap::GenerateBaseMap()
 	// Make the initial platform much much larger so things don't fall off for tests which use a large area (visibility test)
 	Plane->SetActorScale3D(FVector(10000, 10000, 1));
 
-	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, FTransform(FVector(-500, 0, 100)));
+	// Default player start location is chosen so that players spawn on server 1 by default.
+	// Individual test maps can change this if necessary.
+	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, FTransform(FVector(-100, -100, 100)));
```

Comment:
I would prefer if the player start was a bit further away from the origin, but will leave to @razvan-at-improbable to address, as it would probably also need some sort of rotation change.

Reply:
Out of interest why?

Reply:
if a test does not use the player start, I think that's where the spectator pawn is created, through which you will see the world.